notification/slack/usergrouperror.go

```diff
@@ -19,7 +19,7 @@ type userGroupError struct {
 	ScheduleName string
 	Missing      []notification.User
 
-	OnCall string
+	ErrorMsg string
```

Comment:
This field and the `{{.ErrorMsg}}` line can be removed entirely, that was only used as the placeholder for the on call message. The entire template _is_ the error message